backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/database/jpa/NormReferenceDTO.java

```diff
@@ -31,6 +31,9 @@ public class NormReferenceDTO {
   @JoinColumn(name = "norm_abbreviation_id")
   private NormAbbreviationDTO normAbbreviation;
 
+  @Column(name = "norm_abbreviation_raw_value", insertable = false, updatable = false)
+  private String normAbbreviationRawValue;
```

Comment:
Is the introduction of the rawValue aligned with the YAGNI-Principle? The frontend is not using it at this point, right?

Reply:
But I hope we do it with this refactoring too?